packages/verticaltabs/src/react/list.js

```diff
@@ -1,16 +1,43 @@
 import { css } from 'glamor'
 import React from 'react'
+import PropTypes from 'prop-types'
 
+import Collapsible from '@pluralsight/ps-design-system-collapsible'
 import filterReactProps from '@pluralsight/ps-design-system-filter-react-props'
 
+import { useForceCollapsed } from './context.js'
 import stylesheet from '../css/index.js'
 
 const styles = {
-  list: _ => css(stylesheet['.psds-verticaltabs__list'])
+  list: _ => css(stylesheet['.psds-verticaltabs__list']),
+  listCollapsible: _ => css(stylesheet['.psds-verticaltabs__list--collapsible'])
 }
 
-const List = props => (
-  <ul {...filterReactProps(props, { tagName: 'ul' })} {...styles.list()} />
-)
+export const List = props => {
```

Comment:
This is a nitpicky one, but maybe destrcuture collapsed argument and spread props. Your call.